tests/tracking/fluent/test_fluent.py

```diff
@@ -2072,6 +2073,66 @@ def test_set_active_model_env_var(monkeypatch):
     assert _MLFLOW_ACTIVE_MODEL_ID.get() is None
 
 
+def test_set_active_model_public_env_var(monkeypatch):
```

Comment:
Could we test _get_active_model_id_global works when setting this environment variable?
Could we also test by patching `is_in_databricks_model_serving_environment` to true, since that's for databricks serving workflow :)

Reply:
Absolutely! Added!